docs/source/concept_models/vivarium_iv_iron/maternal_model/concept_model.rst

```diff
@@ -516,15 +518,26 @@ Probability of sampling from a given country's hemoglobin distribution using the
     - Explanation
     - Action plan
     - Timeline
+  * - `Shorter than expected duration of pregnancy <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/validation/maternal/model3.5.1/pregnancy%20model.ipynb>`_ (approximately 4 weeks when expected to be >30)
+    - Not sure 
```

Comment:
We have a postpartum duration issue with 3.5.0, do we also have this pregnancy duration issue in 3.5.0, or is it limited to 3.5.1?

Thanks much for this. I'll make tickets or add details to existing ones.

Reply:
Yeah, it's also there for model 3.5.1, I'll make a note of that.

Reply:
I'll look back and see when I think that arose in our model versioning

Reply:
I'll figure it out. Was just asking if you knew off the top of your head. Seems the problems with 3.5.0 and 3.5.1 are postpartum and pregnancy durations being too short and YLDs appearing to not be stratified correctly.

Reply:
Yes, exactly.